modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach error issues
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node. However, this is not possible because when the node is down, the system cannot unmount the volume. A multi-attach error message is reported, for example:
+
+[source,terminal]
+.Example output
+--
+Unable to attach or mount volumes: unmounted volumes=[sso-mysql-pvol], unattached volumes=[sso-mysql-pvol default-token-x4rzc]: timed out waiting for the condition
+Multi-Attach error for volume "pvc-8837384d-69d7-40b2-b2e6-5df86943eef9" Volume is already used by pod(s) sso-mysql-1-ns6b4
+--
+
+.Solution
+.. *Use RWX volumes.*
++
+For most storage solutions, the solution to the multi-attach error is to use ReadWriteMany (RWX) volumes.
++
+.. *Recover or delete the failed node (vSphere).*
```

Comment:
Is this relevant only for vSphere, or are there other scenarios where a cluster administrator might not want to use RWX? If so, this heading could become more general by removing '(vSphere)'. Then, in the sentence below this bullet point, you could say something like, 'For deployments that require the use of RWO volumes...'. Then, add another point or a note about vSphere volumes not supporting RWX.

Reply:
My understanding is that this scenario is specific to vSphere for the reasons you identified. But theoretically, it could apply to any provider that does not support RWX. So I think your suggestion is more useful, given the chance that a deployment might need RWO. I made some updates for broader use. Great suggestion!